### ISSUE TITLE
Ransomware detection

### ISSUE THREAD

**Issue by gelim** [CrowdStrike/detection-container#15]
Hello,

Thanks for this excellent repo, but I'm failing to see if there is any TTP related to ransomware activity that could trigger a detection on falcon Linux sensor?

Regards,
-- Mathieu

**Comment by shawndwells** [comment]
Hi @gelim - this repo is just a subset of events for testing and demos. We could integrate a ransomware demo though. Do you have samples we could look at?

**Comment by shawndwells** [comment]
Found some - https://github.com/fabrimagic72/malware-samples/tree/master/Ransomware

**Comment by gelim** [comment]
Yes for instance, or the opensource https://github.com/tarcisio-marinho/GonnaCry that can be "defused", but still valid to asess any detection logic on the defense side.

**Comment by redhatrises** [comment]
GonnaCry or Satan could work for examples in the container.

**Comment by shawndwells** [comment]
@gelim Many of the cloud team at CrowdStrike are prepping for AWS re:Inforce and there may not be much movement on this for a few weeks. If you're interested & able, patches would be welcome!

**Comment by gelim** [comment]
If I find some code that trigger ransomware detection logic on Linux falcon sensor, I will update here. For the moment it seems there is only event dedicated to ransomware on the Windows platform.

For instance Gonnacry will be prevented by NGAV (` This file meets the File Attribute ML algorithm's high-confidence threshold for malware.`)

**Comment by shawndwells** [comment]
The GonnaCry should do the trick. Their README calls out being written for Linux - https://github.com/tarcisio-marinho/GonnaCry

![image](https://user-images.githubusercontent.com/5713754/128260689-42979468-e005-40fe-ae24-a367ee6ef6d4.png)

**Comment by gelim** [comment]
As I mentioned previously the original binary is getting a prevention with a generic alert.
Additionally a fresh build will not be prevented nor detected.
